test/unit/custom_spec.js

```diff
@@ -107,3 +107,104 @@ describe("custom canvas rendering", function () {
       .catch(done.fail);
   });
 });
+
+describe("alternate document context", function () {
+  const FontFace = global.FontFace;
+
+  let CanvasFactory;
+
+  beforeAll(() => {
+    global.FontFace = function MockFontFace(name) {
+      this.family = name;
+    };
+    CanvasFactory = isNodeJS ? new NodeCanvasFactory() : new DOMCanvasFactory();
```

Comment:
The `DOMCanvasFactory` gets no parameter here, even though that was added above. Is that expected?

Reply:
I've added it